zc_common/remote_resource/test_mixins.py

```diff
@@ -1,5 +1,7 @@
 import json
 
+import datetime
+import dateutil.parser
```

Comment:
Unfortunately, the parser is required for the test mixin to work properly. You'd only need to install the requirement when running tests, though, so you don't necessarily have to add it to the `requirements.txt`, you could just add it to the `circle.yml` file. However, adding `dateutil` to `requirements.txt` won't affect prod at all.

This is one of the downsides of the approach we chose to take with zc_common, which is that we don't force installation of zc_common's dependencies. So you can import zc_common and only use the timezone code and not have to install DRF/json api, for example.

Ideally, we'd split out all the DRF related code into its own repo/project and force the dependencies. So when you install that project, it'd auto-install its dependencies. For now, I'd rather not go through the trouble due to the number of changes being added frequently. I think that's the step we should take sometime in the next few months. Essentially, split up zc_common into:

1. `zc_microservices`: a package containing all our microservice shared code (JSON API, remote resource, etc)
2. `zc_time`: a package containing all our utilities for handling dates and times